prometheus/prometheus.yml

```diff
@@ -10,4 +10,4 @@ scrape_configs:
 - job_name: scylla
   honor_labels: true
   static_configs:
-  - targets: ['127.0.0.1.18:9103']
+  - targets: ['127.0.0.1:9090']
```

Comment:
why 9090? port 9103 is the default  collectd_exporter port, as describe here
https://github.com/scylladb/scylla/wiki/Monitor-Scylla-with-Prometheus-and-Grafana#collectd_exporter